File: Rosmery.Security.UI/src/app/content/pages/user/models/user.model.ts
import { RoleModel } from './role.model';

export class UserModel {
    userId: string;
    userName: string;
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    role: RoleModel;
    roleId: string;
    isActive: boolean;

    fullName(): string{
        return this.firstName + ' ' + this.lastName
    }

    clear(): void {
        this.userId = '';
        this.userName = '';
        this.firstName = '';
        this.lastName = '';
        this.email = '';
        this.phoneNumber ='';
        this.role = new RoleModel();
    }
}

File: Rosmery.Security.API/Models/UserModel.cs
using Rosmery.Security.Identity.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rosmery.Security.API.Models
{
    public class UserModel
    {
        public Guid UserId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string UserName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public Role Role { get; set; }

    }
}
